inaugurator/server/config.py

```diff
@@ -1,2 +1,3 @@
-PORT = 1013
+import pika
+PORT = pika.ConnectionParameters.DEFAULT_PORT
 AMQP_URL = "amqp://guest:guest@localhost:%d/%%2F" % PORT
```

Comment:
The changing of the port is because the RabbitMQ service runs under user (and group) "rabbitmq", so it does not have permissions to bind sockets to ports with numbers smaller than 1024 (as opposed to the RabbitMQ process that was running with root privileges, since it was invoked directly from rabbitmqwrapper.py).